internal/db/dqlite.go

```diff
@@ -191,6 +191,11 @@ func (db *DB) NotifyUpgraded() {
 	}
 }
 
+// SetClusterCert updates the cluster cert.
+func (db *DB) SetClusterCert(clusterCert *shared.CertInfo) {
+	db.clusterCert = clusterCert
```

Comment:
Does this need some locking in case some other routine is reading the old certificate?

Reply:
Instead of going with this approach, I just decided to do what we already do with the `State` struct and pass the daemon's already thread-safe clusterCert function to the `DB` struct. So now whenever the daemon's cluster cert is updated, the db's should be as well as it's calling the same function.

e8b509879cf6d38b6e0b4cd0bc9030e22bbdf815